### PR TITLE
34 Add linting to /server

### PR DESCRIPTION
**Pull request by kajiwara12**

### CODE REVIEW

**Comment by kajiwara12** [comment]
everything is working now! linter is working

**Comment by kajiwara12** [comment]
good to merge

File: server/server.js
Comment (AGI-CEO) [inline server/server.js:1]: These edits seem to indicate the linter / prettier is formatting things as expected. If this is true, then this PR looks good to merge. @kajiwara12 please confirm.

Comment (kajiwara12) [reply server/server.js:1]: yeah i finally got linter working correctly

Comment (gnprwx) [reply server/server.js:1]: Is this PR only linting what's inside the server folder?

**Review by gnprwx — APPROVED** [review]
All good for /server. Next up will be /client.

**Pull request merged**